platform/darwin/src/MGLGeoJSONOptions.h

```diff
@@ -0,0 +1,13 @@
+#import <Foundation/Foundation.h>
+
+@interface MGLGeoJSONOptions : NSObject
```

Comment:
It’s ironic for me to say this in light of #5970, but I think we could get away with a simple NSDictionary and some global string constants instead of a full-blown options object. That’s the way most Cocoa APIs accept options, especially when not all of the options are required (for example, NSAttributedString). Moving to a loosely-typed dictionary also frees us from having to [hard-code default values](https://github.com/mapbox/mapbox-gl-native/pull/6217/files#diff-cbb5293af6f3a871ebb2bf9565fe13fdR10).